db/seeds.rb

```diff
@@ -5,3 +5,6 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 
+
+# merchant1 = Merchant.create!(name: "Billy's Butters")
```

Comment:
Should we leave in these comments?  Or should we just delete them since we never use seeds or should never use seeds anyways